mojo/dart/packages/mojo/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.4.17
+
+  - 48 changes: https://github.com/domokit/mojo/compare/2829804...1484e83
+
+  MOJO_SDK: 1484e831758d05c7af0f27c199d24e3bbdc6383f
```

Comment:
This file is just copied verbtim from https://github.com/domokit/mojo/blob/master/mojo/dart/packages/mojo/CHANGELOG.md .  I don't have any opinion about how this file is managed.  That's something to take up with the authors of this file in mojo.git.

I suspect we could actually remove it from engine.git.  I can investigate that.